packages/projection/src/operators/Mappers/withMint.ts

```diff
@@ -21,13 +22,14 @@ export interface WithMint {
 
 export const withMint = unifiedProjectorOperator<{}, WithMint>((evt) => ({
   ...evt,
-  mint: evt.block.body.flatMap(({ body: { mint }, auxiliaryData }, txIndex) =>
+  mint: evt.block.body.flatMap(({ body: { mint, outputs }, auxiliaryData }, txIndex) =>
     [...(mint?.entries() || [])].map(
       ([assetId, quantity]): Mint => ({
         assetId,
         assetName: Asset.AssetNameLabel.decode(Cardano.AssetId.getAssetName(assetId)),
         assetNameHex: Cardano.AssetId.getAssetName(assetId),
         compactTxId: computeCompactTxId(evt.block.header.blockNo, txIndex),
+        outputs,
```

Comment:
I don't think the asset has to be **minted**. You could probably recreate the datum by just transferring the tokens back to the script:
```Similarly, if the issuer wants the NFTs/FTs to evolve or wants a mechanism to update the metadata, the reference NFT can be locked at the address of a script with arbitrary logic that the issuer decides.```

It means that we need to check **all** outputs in every transaction, not just in the transactions that some assets were **minted**